classification/feature_generation/file_iterator_test.py

```diff
@@ -18,36 +19,17 @@ def to_stamp(x):
     return (x - epoch).total_seconds()
 
 
-def test_file_iterator_2():
-    path = "gs://world-fishing-827/data-production/classification/release-0.1.2/pipeline/output/features/251822362.tfrecord"
```

Comment:
Are we OK with tests not being hermetic and requiring a pull of data from GCS? Is this input data small enough it could be checked in to github?